src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -10,8 +10,10 @@
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableWinRT Condition="'$(PackageTargetFramework)' == 'netcore50'">true</EnableWinRT>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
```

Comment:
I find myself wondering "is this something that really needs to be in ever package? Can't dir.props set IsPartialFacadeAssembly to true ahead of time when TargetGroup is net45, net451, net452, net46, net461, etc?" (TargetGroup.StartsWith('net4')? :smile:).

Would it ever be wrong?  And, if so, is it fewer places that would need to clear it out than here with libraries needing to set it?